ciri/core.c

```diff
@@ -25,29 +25,22 @@ retcode_t core_init(core_t* const core) {
 
   log_info(CORE_LOGGER_ID, "Initializing consensus\n");
   if (iota_consensus_init(&core->consensus, &db_conf,
-                          &core->transaction_requester) != RC_OK) {
+                          &core->node.transaction_requester) != RC_OK) {
     log_critical(CORE_LOGGER_ID, "Initializing consensus failed\n");
     return RC_CORE_FAILED_CONSENSUS_INIT;
   }
 
-  log_info(CORE_LOGGER_ID, "Initializing transaction requester\n");
-  if (requester_init(&core->transaction_requester, &core->consensus.tangle)) {
-    log_critical(CORE_LOGGER_ID, "Initializing transaction requester failed\n");
-    return RC_CORE_FAILED_REQUESTER_INIT;
-  }
-
-  log_info(CORE_LOGGER_ID, "Initializing cIRI node\n");
-  if (node_init(&core->node, core, &core->consensus.tangle,
-                &core->transaction_requester)) {
-    log_critical(CORE_LOGGER_ID, "Initializing cIRI node failed\n");
+  log_info(CORE_LOGGER_ID, "Initializing gossiping node\n");
```

Comment:
can you change `Initializing gossiping node` -> `Initializing node gossip components` , it's sounds as if the node can't keep a secret.

Reply:
Done